README.md

```diff
@@ -1 +1,49 @@
-Sport1Player-Android
+# Sport1 Player
+
+*Built by Applicaster*
+
+**Supports:** *iOS and Android*
+*Current Version: 0.0.21 (Android), 1.6.1 (iOS)*
+
+## About
+
+a player plugin for sport1, wrapper around JWPlayerPlugin
+
+### features and behaviour:
+
+- display the login plugin when needed (if playable is not free or if the user login token is missing for live content)
```

Comment:
An upper case for every first latter in the list

Reply:
fixed